build.gradle

```
@@ -2,7 +2,7 @@ plugins {
 	id 'idea'
 
 	id 'io.spring.dependency-management' version '1.0.11.RELEASE' apply false
```

I think this should be updated to the current spring dependency plugin version
```suggestion
	id 'io.spring.dependency-management' version '1.1.4' apply false
```